Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1208,7 +1236,9 @@ - (void)serverCallback:(BNCServerResponse *)response {
                     [BNCPreferenceHelper setInstallParams:[response.data objectForKey:DATA]];
                 }
             }
-            
+            if ([BNCPreferenceHelper getNeedAppListCheck]) {
+                [self processListOfApps];
+            }
```

Comment:
It gets checked and added in both 'install' and 'open' callbacks, as scans the apps every 4 days or so